contribute.md

```diff
@@ -4,7 +4,7 @@ Take a look at the issues for bugs that you might be able to help fix.
 
 Once your pull request is merged it will be released in the next version.
 
-We are using https://github.com/GaryJones/wordpress-plugin-svn-deploy to deploy the plugin to SVN.
+We are using [https://github.com/bmlt-enabled/bmlt-wordpress-deploy](https://github.com/bmlt-enabled/bmlt-wordpress-deploy) to deploy the plugin to SVN. This is done automatically upon tagging a release. If you wish to tag a release for testing or a beta release make sure the tag name contains the word beta in it as the deployment to wordpress will be cancelled. For ex. `3.9.6-beta`
```

Comment:
You might want to just have a link to the README.md in the bmlt-wordpress-deploy repo.  Too much language here might make it un-maintainable if you have to make a change.

Reply:
good point